pkg/storaged/content-views.jsx

```diff
@@ -724,15 +745,18 @@ export const Block = ({ client, block, allow_partitions }) => {
 function append_logical_volume_block(client, rows, level, block, lvol) {
     let tabs, desc;
     if (client.blocks_ptable[block.path]) {
+        // XXX - just make this a link to the details page for the block device?
```

Comment:
Does this still need to be resolved here, or is that a todo for later?

Reply:
Ah, good catch. I think we need to resolve this. This is for partition tables on logical volumes, which probably nobody is doing and Cockpit offers no way to do it, so it is super rare. It might not even be supported by LVM2. But we need to make sure nothing explodes. (I should add a test.)

Reply:
This now links to the details page for the block.  Puting partition tables on device mapper devices seems to be broken in all kinds of ways.  It is possible to do it and get the corresponding block devices like `/dev/mapper/vgroup0-lvol0p1`, but formatting doesn't really work, and removing the partition table does not remove the device mapper devices...